plugins/polio/js/src/domains/Campaigns/MainDialog/CreateEditDialog.tsx

```diff
@@ -61,13 +63,18 @@ const CreateEditDialog = ({ isOpen, onClose, campaignId }) => {
     const schema = useFormValidator();
     const { formatMessage } = useSafeIntl();
 
-    const classes = useStyles();
+    const classes: Record<string, string> = useStyles();
 
     const handleSubmit = async (values, helpers) => {
         saveCampaign(convertEmptyStringToNull(values), {
             onSuccess: () => {
                 helpers.resetForm();
                 onClose();
+                dispatch(
+                    enqueueSnackbar(
+                        successSnackBar(MESSAGES.defaultMutationApiSuccess),
+                    ),
+                );
```

Comment:
dispatching the snackbar here so the form can be reset and `onClose` called before the snacbar triggers a re-render of the whole table